examples/lya/paramfile.gadget

```diff
@@ -63,9 +52,9 @@ QuickLymanAlphaProbability = 1 # Set to 1.0 to turn dense gas directly into star
 
 SnapshotWithFOF = 1
 WindModel = nowind
-BlackHoleOn = 0
-BlackHoleFeedbackFactor = 0.05
-BlackHoleFeedbackRadius = 0.
-BlackHoleFeedbackRadiusMaxPhys = 0.
-BlackHoleFeedbackMethod = spline | mass
-SeedBlackHoleMass = 5.0e-5
+# if compiled with BlackHole option, add these lines 
```

Comment:
I suppose these blackhole stuff are all optional now too... Do you want to get rid of them while you are still on it?